ethminer/main.cpp

```diff
@@ -283,7 +283,7 @@ class MinerCLI
 
 		string sched = "sync";
 		app.add_set("--cuda-schedule", sched, {"auto", "spin", "yield", "sync"},
-			"Set the scheduler mode", true)
+			"Set the scheduler mode. Default \"sync\"", true)
```

Comment:
I believe CLI11 should print default values (the last argument `true` requests this).

Reply:
Yes. You're right did not notice it's "compact" way to display defaults.